packages/ui/package-lock.json

```diff
@@ -6676,19 +6676,6 @@
       "integrity": "sha512-fBMHxFHypWHa+IRzhLTcpSv9pg8sAyUhMqWkilhp/3M7PQ3pjAufkFGyqp6CSdb7a0u4p8pYL8CqG6ePV5OJRQ==",
       "dev": true
     },
-    "@ytsaurus/interface-helpers": {
-      "version": "0.1.1",
-      "resolved": "https://registry.npmjs.org/@ytsaurus/interface-helpers/-/interface-helpers-0.1.1.tgz",
-      "integrity": "sha512-ihpxHaFc2MIGsXEnjdKy21jQrs/sV/QOCSdTmAwIjKXZ+9qKeYAbpou9HPzGwNJSu7bJMGnrLNxh6jodaYKUww=="
-    },
-    "@ytsaurus/javascript-wrapper": {
-      "version": "0.2.1",
-      "resolved": "https://registry.npmjs.org/@ytsaurus/javascript-wrapper/-/javascript-wrapper-0.2.1.tgz",
-      "integrity": "sha512-h884FY9pSb/Y7BIM/BV0gXlKazqTOOkihHuLIS+eijWyYP1Sebx+/neXHy+ib+CMepeeYoG3AOT9bAewL4n7Wg==",
-      "requires": {
-        "axios": "^0.21.4"
-      }
-    },
```

Comment:
I am sure we don't need this changes and we don't need last commit which revert it.